packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/ModalForm/index.js

```diff
@@ -114,7 +113,7 @@ const ModalForm = ({ onSuccess, onToggle }) => {
     <ModalLayout onClose={onToggle} labelledBy="title">
       <ModalHeader>
         <Breadcrumbs label={headerTitle}>
-          <Crumb>{headerTitle}</Crumb>
+          <Crumb isCurrent>{headerTitle}</Crumb>
         </Breadcrumbs>
```

Comment:
What do you think about adding a TODO? Technically this is not a breadcrumb navigation and for screenreader users this might be very confusing.

Reply:
Yeah I agree, to be honest i have a major issue with how we use modals and breadcrumbs, they're never the title for the modal imo.